.gitignore

```diff
@@ -96,3 +96,10 @@ temp
 /packages/*/*/test-results
 /packages/*/*/playwright-report
 
+packages/tools/lint-package/src/helpers.js
+packages/tools/lint-package/src/index.js
+packages/tools/lint-package/src/logger.js
+packages/tools/lint-package/src/rules/api-extractor/types.js
+packages/tools/lint-package/src/rules/eslint/config.js
+packages/tools/lint-package/src/rules/package-json/scripts.js
+packages/tools/lint-package/src/rules/typescript/extends.js
```

Comment:
Would it be better to make a separate PR for this?

Is this happening when running `tsc` in that package? It should, based on tsconfig, create a `lib` for files like this.

Do we maybe need a more generic one? Something like

```
packages/*/*/src/**/*.js
```

Reply:
I don't know how they came up here